README.md

```diff
@@ -111,6 +111,30 @@ Connecting an SMT with an RMT this way is not supported.
 
 Available configuration options can be found in the `etc/rmt.conf` file.
 
+The recommended way to perform initial configuration is [YaST RMT module](https://github.com/SUSE/yast2-rmt). The RPM package is [available on OBS](https://build.opensuse.org/package/show/systemsmanagement:SCC:RMT/yast2-rmt).
```

Comment:
"configuration is using the"

The package is included in sle15 too though, isn't it?

Reply:
It is, but we typically have more recent builds in OBS and this is mostly for development/pre-SLES15 testing purposes.